<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fen_AG</class>
 <widget class="QDialog" name="Fen_AG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>670</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>772</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Paramètres</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre de la population</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Génération de la population initiale</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_5">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Méthode Aléatoire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greedy Aléatoire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mixte</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Taux de population intérmédiaire</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Probabilité de croisement</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Probabilité de mutation</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Stratègie de sélection</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Aléatoire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Elitiste</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Roue de la fortune</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Stratègie de remplacement</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>Elitiste</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50/50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Générationelle</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Méthode de mutation</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Suppression</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Addition</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Echange</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Méthode de croisement</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>1 pt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 pts</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Critère d'arrêt</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>Nombre d'itérations</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ecart Maximal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Taux d'évolution</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Nombre d'itérations</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Ecart Maximal</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Taux d'évolution</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Seuil de réparation</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_13">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Ratio (Si initiale mixte)</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QSpinBox" name="spinBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>375</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Approche</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Intensification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Diversification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Mixte</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>660</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Sélectionner un Benchmark</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Charger des paramètres prédéfinis</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Fen_AG</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Fen_AG</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
